### ISSUE TITLE
Is that correct the  effect dependencies are not exhaustive

### ISSUE THREAD

**Issue by soulcm** [vercel/swr#263]
In the [use-swr](https://github.com/zeit/swr/blob/master/src/use-swr.ts) file, I found some effect dependencies not exhaustive, I want to konw it is correct or not.

![image](https://user-images.githubusercontent.com/5706155/74588343-af6f8980-5036-11ea-9ee3-8550e0820aec.png)
If add eslint-plugin-react-hooks, and  add ```"react-hooks/exhaustive-deps": "warn"``` in your ```.eslintrc```  It will warn that ```React Hook useEffect has missing dependencies: 'config. suspense'```

the same as below code
```
const revalidate = useCallback(() => {
  // ... 
if (fnArgs !== null) {
            CONCURRENT_PROMISES[key] = fn(...fnArgs)
          } else {
            CONCURRENT_PROMISES[key] = fn(key)
          }
// ...
}, [key]) // the  fn args also not add the dependencies
```
anybody could make sense for me is that conrrect or not.  thanks a lot

**Comment by koba04** [comment]
I've created a PR to fix this.

**Comment by koba04** [comment]
This has been fixed via #886, so we could close this.

**Comment by huozhi** [comment]
Close this as the warning has been disabled in latest commits. `config.suspense` is required to be static during the swr lifecycle.
